coordinator/src/routes/prescription/post-send-message.ts

```diff
@@ -3,6 +3,7 @@ import * as translator from "../../services/translation/translation-service";
 import {Bundle} from "../../model/fhir-resources";
 import {sendData} from "../../services/spine-communication";
 import Hapi from "@hapi/hapi";
+import * as common from "../../services/translation/common"
```

Comment:
seems sensible to import the `wrapInOperationOutcome` function the same way that `sendData` is imported on line 4